src/btchip_transaction.c

```diff
@@ -1094,6 +1094,11 @@ void transaction_parse(unsigned char parseMode) {
                     }
 
                     if (btchip_context_D.NU5Transaction) { 
+                        // Only expiryHeight should remain at this point
```

Comment:
I would have a detailed comment explaining why you have to check that.

Reply:
Changed to : 
`// We don't support sapling or orchard actions 
// Only expiryHeight should remain at this point`